### PR TITLE
Add CreateMultisig RPC to stakepoold

### PR DESCRIPTION
**Pull request by jholdstock**
- Move CreateMultisig to stakepoold RPC. 
- Add extra fields to WalletInfo on stakepoold RPC 
- Remove Ping method from the RPC, it wasnt being used at all
- Remove a bunch of unused code in dcrclient.go
#227

### CODE REVIEW

File: backend/stakepoold/rpc/rpcserver/server.go [hunk 1]
```diff
@@ -33,8 +33,8 @@ const (
 	// collection cycle to also trigger a timeout but the current allocation
 	// pattern of stakepoold is not known to cause such conditions at this time.
 	GRPCCommandTimeout = time.Millisecond * 100
-	semverString       = "5.0.0"
-	semverMajor        = 5
+	semverString       = "7.0.0"
```
Comment (JoeGruffins) [inline backend/stakepoold/rpc/rpcserver/server.go]: Did you mean to increment this by two?

Comment (JoeGruffins) [reply backend/stakepoold/rpc/rpcserver/server.go]: I guess I was looking at a cached version. It was already changed before I wrote this. Srry.

Comment (jholdstock) [reply backend/stakepoold/rpc/rpcserver/server.go]: No worries, I think you might have been reading it as I was making the change :)

**Comment by jholdstock** [comment]
Thanks @JoeGruffins, I have implemented all of your suggestions.

**Comment by JoeGruffins** [comment]
Works for me!

**Pull request merged**